build.gradle.kts

```diff
@@ -0,0 +1,64 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+	repositories {
+		google()
+		mavenCentral()
+		maven("https://maven.fabric.io/public")
+		maven("https://maven.google.com")
+	}
+	dependencies {
+		classpath("com.android.tools.build:gradle:7.0.3")
+		classpath(kotlin("gradle-plugin", version = "1.5.31"))
+		classpath(kotlin("serialization", version = "1.5.31"))
+		classpath("com.google.gms:google-services:4.3.10")
+		classpath("com.google.firebase:firebase-crashlytics-gradle:2.8.0")
+		// NOTE: Do not place your application dependencies here; they belong
```

Comment:
Cleanup Comments.

Reply:
This seems to be a comment left by android studio when generating the original files, removed anyways